packages/cli/src/generators/typescript/analytics-generator.ts

```diff
@@ -0,0 +1,266 @@
+import { CodeBlock, CodeGenerator } from "../code-generator";
```

Comment:
Minor, seem using the mix file naming convention, `xxx-yyy` and `XxxYyy`, maybe update to a unified style?

Reply:
Thanks @liuyang1520 for naming files I use the pattern:
* For single class export file name is Pascal case of the export
* For files with multiple exports I use kebab case

WDYT? 

For the alpha I am keeping some functionally similar classes in the same file to be have flexible structuring. Once the code base is more stable I plan to break these up into more focused classes etc.